resource/filter.go

```diff
@@ -167,3 +177,33 @@ func (f Filter) matches(r *Resource) bool {
 	}
 	return false
 }
+
+func (f *StringFilter) matches(s string) (matched bool, err error) {
+	ok, err := regexp.MatchString(f.Pattern, s)
+	if err != nil {
+	  return false, err
+	}
+	
+	if f.Negate {
+	  return !ok, nil
+	}
+	
+	return ok, nil
+	
+		ok = !ok
```

Comment:
Ah, there is some code left that my suggestion didn't replace. Also, can you please rebase? There are some conflicts.

Reply:
Fixed